sdk/node/test/receivers.js

```diff
@@ -0,0 +1,127 @@
+
+/* eslint-env mocha */
+
+const chain = require('../dist/index.js')
+const uuid = require('uuid')
+const assert = require('assert')
+const chai = require('chai')
+const chaiAsPromised = require('chai-as-promised')
+
+chai.use(chaiAsPromised)
+const expect = chai.expect
+
+const client = new chain.Client()
+const signer = new chain.HsmSigner()
+
+function createAccount() {
+  return client.mockHsm.keys.create()
+    .then((key) => {
+      signer.addKey(key, client.mockHsm.signerConnection)
+      return client.accounts.create({
+        alias: `account-${uuid.v4()}`,
+        rootXpubs: [key.xpub],
+        quorum: 1
+      })
+    })
+}
+
+function createAsset() {
```

Comment:
I bet that future tests may want to use these too. No need to change it now, but think about extracting these to imported functions if the next set of tests use similar helpers.